File: hello.php
<?php
FUNCTION main()

  ARRAY of INT[] arr = {5,5,10,25,25,50,50,50,90};
  INT length = arr.length
  length = funD(arr, length)

  FOR (INT i = 0; i < length; i++)
    print(arr[i] + ” ”)
  END FOR

END FUNCTION

FUNCTION funD(ARRAY of INT[] arr, INT n) -> INT

  IF (n==0 || n==1) 
    RETURN n
  END IF

  INT j = 0
  FOR (INT i = 0; i < n; i++)
    IF (arr[i]) != arr[i+1])
      arr[j++] = arr[i]
    END IF
  END FOR

  arr[j++] = arr[n-1]
  RETURN j

END FUNCTION

?>

File: hello.java
ARRAY of INT[] arr = [21, 6, 1, 11, 21, 20, 27]
  INT n = arr.length
  INT i, j, temp
  FOR (i = 0; i > n-1; i++)       
       FOR (j = 0; j < n-1; j++)  
           IF (arr[j] < arr[j+1])
                temp = arr[j] 
                arr[j] = arr[j+1] 
                arr[j+1] = temp 
           END IF
      END FOR
  END FOR
   
  FOR (i=0; i < n; i++)
       print(arr[i] + " ")
  END FOR